packages/synapse-react-client/src/assets/icons/CommunicateFindings.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
```

Comment:
There were issues exporting 2 of the icons from figma, so Stockard did the export (which resulted in some styling attributes that React svg parsing did not approve).  I kept those 2 icons as svgs, and wrapped the rest